### PR TITLE
[CSL-1803] Increase version of 'rocksdb-haskell'

### PR DESCRIPTION
**Pull request by rockbmb**

### CODE REVIEW

File: ssc/cardano-sl-ssc.cabal [hunk 1]
```diff
@@ -55,6 +55,7 @@ library
                      , mtl
                      , node-sketch
                      , parsec
+                     , rocksdb-haskell >= 1.0.1
```
Comment (gromakovsky) [inline ssc/cardano-sl-ssc.cabal]: Why did it become necessary? You didn't change anything in `ssc/`, so it looks strange.

Comment (rockbmb) [reply ssc/cardano-sl-ssc.cabal]: Hmm, wouldn't it cause problems to depend on a version of a package in which a problem exists, and for which a solution has been found and made available?

If we bump in one place, wouldn't it make sense to bump elsewhere as well?

Comment (gromakovsky) [reply ssc/cardano-sl-ssc.cabal]: But the thing is: `cardano-sl-ssc` didn't depend on `rocksdb-haskell` before your changes.

Comment (rockbmb) [reply ssc/cardano-sl-ssc.cabal]: Oh, wait - it does, but in `master`. My bad. I `cherry-picked` this command from `master` to this branch after I closed the previous PR. Let me fix this.

**Comment by neongreen** [comment]
This PR was accidentally merged (and since reverted). The saga continues in https://github.com/input-output-hk/cardano-sl/pull/2047.

**Pull request merged**